src/base.mli

```diff
@@ -101,3 +101,37 @@ val available : unit -> bool
 [@@@js.implem
 let available () =
   Js_of_ocaml.Js.Optdef.test Js_of_ocaml.Js.Unsafe.global##.mixpanel]
+
+val set_group :
+  group_key:string ->
+  group_ids:string list ->
+  ?callback:(Ojs.t -> unit) ->
+  unit ->
+  unit
+  [@@js.global "mixpanel.set_group"]
+
+module Group : sig
+  type group_id
+
+  (*Pas sûr de si c'est bien des properties*)
+  val get_group : group_key:string -> group_id:Ojs.t -> group_id
+    [@@js.global "mixpanel.get_group"]
+
+  val set :
+    group_id ->
+    prop:Properties.t ->
+    ?to_:Ojs.t ->
```

Comment:
```suggestion
```

Since, from my understanding of the [documentation](https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelgroupset), this argument only makes sense if the argument `prop` is a string.

Reply:
Indeed, the "Property.t" form already apply the function of "to_".